pom.xml

```diff
@@ -21,7 +21,7 @@
     </licenses>
 
     <properties>
-        <jenkins.version>1.625.3</jenkins.version>
+        <jenkins.version>1.642.3</jenkins.version>
```

Comment:
96% of all users of the last release (2.2.3) are on 2.46 or higher of Jenkins....
http://stats.jenkins.io/pluginversions/github-branch-source.html

Reply:
In this case, I have upgraded to the minimal version to maintain the alignment between the plugin's baseline and and the ones from its dependencies. I think increasing the baseline even more is a decision that probably corresponds to the maintainer